lessons/session_6/session_6_deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+marp: true
+theme: uncover
+_class: invert
+paginate: true
+output: pdf_document
+---
+
+# ML Systems in Production
+# Session 6: Deployment
+Jesús A. Calderón
+February 2024
+
+---
+
+# Agenda
+
+**6.1Model Deployment and Prediction Service **
+	
++ ML Deployment Myths
++ Batch Prediction vs Online Prediction
++ Model Compression
++ ML in the Cloud
+	
+**Assignment**
+
++ Connect to Power BI
```

Comment:
The usage of Power BI requires that all learners to have a Microsoft account, that could be problematic as we have also asked learners to create google accounts early on in the Session. Not something regarding your PR, but something to note for the admin team.

Reply:
I would not expect that a MS account is necessary. We can connect directly using the desktop version. This is not a BI course, so I would not go into the details of connecting to the Fabric Platform nor maintaining a production pipeline there. The PBI Desktop version is free.